compiler/src/yul/mappers/expressions.rs

```diff
@@ -62,6 +62,27 @@ pub fn expr(
     }
 }
 
+
+pub fn expr_bin_operation<'a>(exp: &Spanned<vyp::Expr<'a>>) -> Result<ExtExpression, CompileError> {
+    if let vyp::Expr::BinOperation{ left, op, right } = &exp.node {
+        // FIXME: How do I get what I need for the `add`
+        let yul_left = literal_expression! {(left.node)};
+        let yul_right = literal_expression! {(right.node)};
+        return match op.node {
+            vyp::BinOperator::Add => {
+                Ok(ExtExpression {
+                    expression: expression! { add([yul_left], [yul_right]) },
```

Comment:
@g-r-a-n-t Please ignore all the sloppiness in this code. Can you poke me in the right direction how to construct the `add(?, ?)` here? I need to get to "whats behind" the left and right nodes but I'm a bit lost how exactly.